spec/requests/serialization_spec.rb

```diff
@@ -72,4 +72,26 @@
     expect(Incident.first).to be_valid
     expect(Incident.first).to be_in_review
   end
+
+  it 'can create an incident via XML upload', driver: :poltergeist do
```

Comment:
Ah, good point. I'll add some, though I'm not sure if I'll be able to cover all corner cases given how complicated XML is.